data/org.sugarlabs.gschema.xml

```diff
@@ -333,4 +333,177 @@
             <description>This string is displayed in the control panel, about computer section.</description>
         </key>
     </schema>
+    <enum id='org.sugarlabs.desktop.GDesktopProxyMode'>
```

Comment:
This is a very nicely documented schema, but I think that something lost the indentation up the top here.